src/main.cpp

```diff
@@ -21,45 +21,58 @@ enum StatusCode {
   SDL_INIT_FAILURE
 };
 
-const char *status_messages[] = {
+const char* status_messages[] = {
```

Comment:
pointer and reference indicator should be attached to the name, i.e. it was good, now bad.